src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -74,178 +80,150 @@ limitations under the License.
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="1"/>
-
     <features>
       <feature bit="0" code="SN" name="SceneNames" summary="The ability to store a name for a scene.">
         <optionalConform/>
       </feature>
     </features>
-    <attribute side="server" code="0x0000" name="LastConfiguredBy" define="LAST_CONFIGURED_BY" type="node_id" isNullable="true" optional="true">
-      <optionalConform/>
-    </attribute>
-    <attribute side="server" code="0x0001" name="SceneTableSize" define="SCENE_TABLE_SIZE" type="int16u" min="16" default="16">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="0x0002" name="FabricSceneInfo" define="FABRIC_SCENE_INFO" type="array" entryType="SceneInfoStruct">
-      <mandatoryConform/>
-    </attribute>
-    
+    <attribute side="server" code="0x0000" name="SceneTableSize" define="SCENE_TABLE_SIZE" type="int16u" default="16">SceneTableSize</attribute>
+    <attribute side="server" code="0x0001" name="FabricSceneInfo" define="FABRIC_SCENE_INFO" type="array" entryType="SceneInfoStruct">FabricSceneInfo</attribute>
     <command source="client" code="0x00" name="AddScene" response="AddSceneResponse" isFabricScoped="true" optional="false" cli="chip scenes add">
-      <description>
-        Add a scene to the scene table. Extension field sets are supported, and are inputed as '{"ClusterID": VALUE, "AttributeValueList":[{"AttributeID": VALUE, "Value*": VALUE}]}'
-      </description>
-      <arg name="GroupID" type="group_id"/>
-      <arg name="SceneID" type="int8u"/>
-      <arg name="TransitionTime" type="int32u"/>
-      <arg name="SceneName" type="char_string"/>
-      <arg name="ExtensionFieldSets" type="ExtensionFieldSet" array="true"/>
+      <description>It is not mandatory for an extension field set to be included in the command for every cluster on that endpoint that has a defined extension field set.</description>
```

Comment:
This is ... a pretty terrible description.  Followup: get this fixed.

Reply:
Created an issue here: https://github.com/project-chip/connectedhomeip/issues/38033
In the mean time, could we revert to the previous definition?